.sequelizerc

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
```

Comment:
You need to require dotenv in this file for the app, I would love for you to find out why dotenv is required in this file and what breaks if you don't require it.

Reply:
Ya nice one. I did not test it at first. The dotenv is required there to help sequelize have access to our environment variables. For instance, in our config.js file where we configured our sequelize we used something like this `username: process.env.DB_USER,` so that env that is required there will help sequelize have access to whatever that is stored in `process.env.DB_USER`